cylc/flow/option_parsers.py

```diff
@@ -43,6 +43,12 @@
     setup_segregated_log_streams,
 )
 
+WORKFLOW_ID_ARG_DOC = ('WORKFLOW', 'Workflow ID')
+WORKFLOW_ID_MULTI_ARG_DOC = ('WORKFLOW ...', 'Workflow ID(s)')
+WORKFLOW_ID_OR_PATH_ARG_DOC = ('WORKFLOW | PATH', 'Workflow ID or path')
+WORKFLOW_ID_OR_NAME_ARG_DOC = ('WORKFLOW', 'Workflow ID or name')
+ID_MULTI_ARG_DOC = ('ID ...', 'Workflow/Cycle/Family/Task ID(s)')
+RELATIVE_ID_MULTI_ARG_DOC = ('ID ...', 'Cycle/Family/Task ID(s)')
```

Comment:
This isn't actually a relative ID (try `cylc hold cycle/task` for example) it's a regular ID (i.e. `workflow//cycle/task`) however, the value must be task-like (i.e. `w//c`, `w//c/t`) rather than workflow like (i.e. `w//`).

So it's kinda more like:

```suggestion
ID_MULTI_ARG_NO_WORKFLOW_DOC = ('ID ...', 'Cycle/Family/Task ID(s)')
```

Reply:
FULL would include jobs but I don't mind.